test/state/host.cpp

```diff
@@ -478,11 +478,7 @@ bytes32 Host::get_block_hash(int64_t block_number) const noexcept
     if (const auto& it = m_block.known_block_hashes.find(block_number);
         it != m_block.known_block_hashes.end())
         return it->second;
-
-    // Convention for testing: if the block hash in unknown return the predefined "fake" value.
-    // https://github.com/ethereum/go-ethereum/blob/v1.12.2/tests/state_test_util.go#L432
-    const auto s = std::to_string(block_number);
-    return keccak256({reinterpret_cast<const uint8_t*>(s.data()), s.size()});
+    return {};
```

Comment:
The line looks to be covered, so still used somewhere?

Reply:
The coverage comes from ethereum/tests. I'll need to check but IIRC there were some random tests having `BLOCKHASH`.

Reply:
This test has it: https://github.com/ethereum/legacytests/blob/master/Constantinople/GeneralStateTests/stRandom/randomStatetest101.json.

Reply:
I guess value is not used in it then